### DISCUSSION TITLE
Does Trivy scan docker volume plugins

### DISCUSSION THREAD

**Discussion by falcanta-akamai** [aquasecurity/trivy#2909] [Q&A]
Trivy can scan docker images, but is trivy capable of scanning docker volume [plugins](https://docs.docker.com/engine/extend/plugins_volume/)? I didn't see anything in the trivy documentation that indicated it could scan docker volume plugins.

**Comment by AnaisUrlichs** [comment]
Heya, I have never used Docker volume plugins but it looks like you are adding data to a running image 
Trivy treats resources as static e.g. a container image at rest -- thus, it would not detect any resources that are added once you run the container

However, the Docker documentation is mentioning Amazon EBS as a potential solution -- while Trivy container image scans cannot scan volumes added to a container image -- since the latest release, you can scan Amazon EBS for security issues https://github.com/aquasecurity/trivy/discussions/3231
